tests/foreman/cli/test_bootstrap_script.py

```diff
@@ -5,9 +5,8 @@
 
 
 class BootstrapScriptTestCase(CLITestCase):
-    """Test class for bootstrap scipt."""
+    """Test class for bootstrap script."""
 
-    @stubbed()
     @classmethod
     def setUpClass(cls):
         """ create VM for testing """
```

Comment:
Ok, if it's unstubbed - it's also required to call `super()` here. And `# TODO: ` comment pointing that `setUpClass()` isn't actually fully implemented and needs to be updated would be useful too.